pkg/kubelet/kubelet.go

```diff
@@ -90,6 +90,9 @@ const (
 
 	// system default DNS resolver configuration
 	ResolvConfDefault = "/etc/resolv.conf"
+
+	// Period for performing global cleanup tasks.
+	housekeepingPeriod = time.Second * 2
```

Comment:
I am more conservative on adding flags since people would start depending on it :)
We didn't have this housekeeping period knob previously, perhaps we can wait a little bit until it's stabilized and users want this level of control?